pkg/system/utils_unix.go

```diff
@@ -9,7 +9,8 @@ import (
 	"strings"
 )
 
-func resolveBinary(binname string) (string, error) {
+// ResolveBinary resolves a binary path (synlinks and such).
+func ResolveBinary(binname string) (string, error) {
```

Comment:
I believe with the suggested split up  this won't need to be exported anymore, as the only consumers will be in `pkg/system/user`